pkg/manifests/config.go

```diff
@@ -404,21 +394,7 @@ func NewDefaultUserWorkloadMonitoringConfig() *UserWorkloadConfiguration {
 }
 
 // IsUserWorkloadEnabled checks if user workload monitoring is
-// enabled. If old techPreview configuration exists, we warn user in logs.
+// enabled.
```

Comment:
Can we entirely remove this func as it is simply the lookup of one value and in go we normally avoid writing getters/setters?